cmd/helm/install.go

```diff
@@ -58,6 +75,13 @@ func runInstall(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func vals() ([]byte, error) {
+	if installValues == "" {
+		return []byte{}, nil
+	}
+	return ioutil.ReadFile(installValues)
```

Comment:
does this assume an absolute path? if so, should probably note in installValues description, e.g.  "absolute path to a values TOML file"